src/devices/Uln2003/Uln2003.cs

```diff
@@ -63,7 +65,8 @@ public class Uln2003 : IDisposable
         /// <param name="pin4">The GPIO pin number which corresponds pin D on ULN2003 driver board.</param>
         /// <param name="controller">The controller.</param>
         /// <param name="shouldDispose">True to dispose the Gpio Controller</param>
-        public Uln2003(int pin1, int pin2, int pin3, int pin4, GpioController controller = null, bool shouldDispose = true)
+        /// <param name="stepsToRotate">Amount of steps needed to rotate motor once in HalfStepMode.</param>
+        public Uln2003(int pin1, int pin2, int pin3, int pin4, GpioController controller = null, bool shouldDispose = true, int stepsToRotate = 4096)
```

Comment:
Thanks to new parameter we could use other step motors with different amount of steps needed to rotate.